core/src/main/kotlin/com/github/quillraven/darkmatter/ui/MenuUI.kt

```diff
@@ -0,0 +1,63 @@
+package com.github.quillraven.darkmatter.ui
+
+import com.badlogic.gdx.scenes.scene2d.ui.ImageButton
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton
+import com.badlogic.gdx.utils.Align
+import com.badlogic.gdx.utils.I18NBundle
+import ktx.i18n.get
+import ktx.scene2d.KTableWidget
+import ktx.scene2d.imageButton
+import ktx.scene2d.label
+import ktx.scene2d.scene2d
+import ktx.scene2d.table
+import ktx.scene2d.textButton
+
+
+private const val OFFSET_TITLE_Y = 15f
+private const val MENU_ELEMENT_OFFSET_TITLE_Y = 20f
+private const val MENU_DEFAULT_PADDING = 5f
+
+class MenuUI(bundle: I18NBundle) {
+    val table: KTableWidget
+    lateinit var startGameButton: TextButton
+    lateinit var soundButton: ImageButton
+    lateinit var highScoreButton: TextButton
+    lateinit var quitGameButton: TextButton
```

Comment:
Refresh the dependencies and try to replace `lateinit var` with `val`. I added Kotlin contracts to the project.

Reply:
I am failing to get that SNAPSHOT again :D Makes me crazy ... tried --refresh-dependencies, cache clearing and other things but I don't get the changes :(

Reply:
Try deleting `~/.gradle/caches/modules-2/files-2.1/io.github.libktx`.

Reply:
I'll upload a stable release over the weekend, but I'd appreciate it if you checked if contracts work correctly first.

Reply:
trying to make it work haha ... I tried the button below and now everything is messed up. Unresolved reference debug, error for LOG calls, receiver and block for ashley, etc...

Will try deleting the cache now and reimport everything. There are two things that I hate most in programming:
1) buggy build tools where unfortunately both maven and gradle have their problems
2) developing UI because even if I understand the theory I never get the result that I expect (looking at you html tables and scene2d table ;))

Reply:
If you're desperate, delete Gradle caches (possibly only from the folder that I listed in the last comment) and idea folder with project setup. Then reimport the project and it should work as expected.

I'll wait with the release for your feedback. Thanks.

Reply:
It is working now! Will do a new commit soon. And you can be proud - you made it into the credits of this AAA billion euro super famous game ;)